app/config/services.php

```diff
@@ -2,193 +2,209 @@
 use Phalcon\Mvc\View;
 use Phalcon\Crypt;
 use Phalcon\Mvc\Dispatcher;
-use Phalcon\Mvc\Url as UrlResolver;
-use Phalcon\Db\Adapter\Pdo\Mysql as DbAdapter;
+use Phalcon\Url as UrlResolver;
+use Phalcon\Db\Adapter\Pdo\Mysql as Connection;
 use Phalcon\Mvc\View\Engine\Volt as VoltEngine;
 use Phalcon\Mvc\Model\Metadata\Files as MetaDataAdapter;
-use Phalcon\Session\Adapter\Files as SessionAdapter;
+use Phalcon\Mvc\Model\Manager as ModelsManager;
+use Phalcon\Session\Manager;
+use Phalcon\Session\Adapter\Files;
 use Phalcon\Flash\Direct as Flash;
-use Phalcon\Logger\Adapter\File as FileLogger;
-use Phalcon\Logger\Formatter\Line as FormatterLine;
+use Phalcon\Flash\Session as FlashSession;
 use Vokuro\Auth\Auth;
 use Vokuro\Acl\Acl;
-use Vokuro\Mail\Mail;
+//use Vokuro\Avatar\Gravatar;
 
 /**
- * Register the global configuration as config
- */
+* Register the global configuration as config
+*/
 $di->setShared('config', function () {
-    $config = include APP_PATH . '/config/config.php';
-    
-    if (is_readable(APP_PATH . '/config/config.dev.php')) {
-        $override = include APP_PATH . '/config/config.dev.php';
-        $config->merge($override);
-    }
-    
-    return $config;
+  $config = include APP_PATH . '/config/config.php';
+  if (is_readable(APP_PATH . '/config/config.dev.php')) {
+    $override = include APP_PATH . '/config/config.dev.php';
+    $config->merge($override);
+  }
+  return $config;
+});
+
+
+/**
+* add prefix
+*/
+/*
+$di->set("modelsManager", function () {
+$modelsManager = new ModelsManager();
+$modelsManager->setModelPrefix("vk_");
+return $modelsManager;
```

Comment:
what is it

Reply:
If anyone needs a prefix for the mysql table.
Example: SELECT * FROM `vk_users`

Reply:
My  proposal is to create a docblock, instead of leaving here a piece of commented code that looks like you forgot to delete it